### PR TITLE
words: add word we forgot

### PR DESCRIPTION
**Pull request by bradfitz**
I explained this tails/scales list to my 5yo and he looked at me like it was the most obvious idea ever. Of course we'd make such lists at work!  What else do grown-ups do all day? And then he wouldn't stop talking about coelacanths and I had no clue what he was saying or how to spell it until I asked my phone and the phone apparently understood me and I realized it was a fish and he was helping me? I think?

### CODE REVIEW

File: words/scales.txt [hunk 1]
```diff
@@ -318,3 +318,4 @@ uaru
 vimba
 wahoo
 zebra
+coelacanth
```
Comment (bradfitz) [inline words/scales.txt:321]: or do fish go in both files, @catzkorn?

Comment (catzkorn) [reply words/scales.txt:321]: If it has a tail and it has scales, it goes in both!

**Review by catzkorn — APPROVED** [review]
This has made my day ♥️

File: words/scales.txt (hunk 1 above)
Comment (bradfitz) [reply words/scales.txt:321]: Done.

**Pull request merged**